frontend/src/main/pages/MyReviews/MyReviewsIndexPage.js

```diff
@@ -1,17 +1,47 @@
+import React from "react";
+import { useBackend } from "main/utils/useBackend";
+
 import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
+import ReviewsTable from "main/components/Reviews/ReviewsTable";
+import { useCurrentUser, hasRole } from "main/utils/currentUser";
+import { Button } from "react-bootstrap";
 
 export default function MyReviewsIndexPage() {
-  // Stryker disable all : placeholder for future implementation
+  const currentUser = useCurrentUser();
+
+  const createButton = () => {
+    if (hasRole(currentUser, "ROLE_USER")) {
+      return (
+        <Button
+          variant="primary"
+          href="/myreviews/create"
+          style={{ float: "right" }}
+        >
+          Create Review
+        </Button>
+      );
+    }
+  };
```

Comment:
Can we remove this?   It doesn't seem to have a purpose.  Reviews are created by the `Review Item` button on the MenuItemTable.   It doesn't make sense to create a review except from that page.  (There's nothing to review.)